timbermill-java/timbermill-local/src/main/java/com/datorama/oss/timbermill/cron/CronsRunner.java

```diff
@@ -122,4 +125,18 @@ private void runPartialMergingTasksCron(ElasticsearchClient es, String mergingCr
 		scheduler.scheduleJob(job, trigger);
 	}
 
+	private void runIndexMergingTasksCron(ElasticsearchClient es, RedisService redisService, String indexMergingCronExp) throws SchedulerException {
+		JobDataMap jobDataMap = new JobDataMap();
+		jobDataMap.put(CLIENT, es);
+		jobDataMap.put(REDIS_SERVICE, redisService);
+		JobDetail job = newJob(IndexMergerJob.class)
+				.withIdentity("job7", "group7").usingJobData(jobDataMap)
```

Comment:
Is it possible to have a more meaningful identity?

Reply:
did the same as all the others